rn/Teacher/src/modules/profile/Profile.js

```diff
@@ -202,7 +202,6 @@ export class Profile extends Component<Object, State> {
         Linking.openURL('https://community.canvaslms.com/community/answers/guides/mobile-guide/content?filterID=contentstatus%5Bpublished%5D~category%5Btable-of-contents%5D')
         break
       case 'terms':
-        await this.props.navigator.dismiss()
```

Comment:
We already dismissed the profile drawer when the settings action sheet was shown.